themes/theme-ios11/pages/Browse/components/SearchField/index.jsx

```diff
@@ -134,13 +134,13 @@ class SearchField extends Component {
    * @param {Object} e The event object.
    * @param {string} [query] Defaults to query in state.
    */
-  handleSubmit = (e, query) => {
+  handleSubmit = (e, query = this.state.query) => {
```

Comment:
The default value is now only applied when `query` is `undefined`. Before it was also used when the parameter value was `""` or `null`,